sql/_05_plcsql/_01_testspec/_02_declaration/_01_parameter/_03_type_conversion/cases/02_01_03-13_sql_to_param_CHAR.sql

```diff
@@ -0,0 +1,208 @@
+end;
+call t_CHAR_CHAR('CHAR', 'CHAR', 'CHAR CUBRID' ) ;
+drop procedure t_CHAR_CHAR ;
+
+
+call print_message('t_VARCHAR_CHAR. This scenario is a success.');
+create or replace procedure t_VARCHAR_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_VARCHAR_CHAR('VARCHAR', 'CHAR', 'VARCHAR CUBRID' ) ;
+drop procedure t_VARCHAR_CHAR ;
+--BUG
+
+call print_message('t_SET_CHAR. This scenario is a failure.');
+create or replace procedure t_SET_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_SET_CHAR('SET', 'CHAR', {'c','c','c','b','b','a'} ) ;
+drop procedure t_SET_CHAR ;
+--BUG
+
+call print_message('t_MULTISET_CHAR. This scenario is a failure.');
+create or replace procedure t_MULTISET_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_MULTISET_CHAR('MULTISET', 'CHAR', {'c','c','c','b','b','a'} ) ;
+drop procedure t_MULTISET_CHAR ;
+--BUG
+
+call print_message('t_LIST_CHAR. This scenario is a failure.');
+create or replace procedure t_LIST_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_LIST_CHAR('LIST', 'CHAR', {'c','c','c','b','b', 'a'} ) ;
+drop procedure t_LIST_CHAR ;
+--BUG
+
+call print_message('t_ENUM_CHAR. This scenario is a failure.');
+create or replace procedure t_ENUM_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_ENUM_CHAR('ENUM', 'CHAR', 'yellow' ) ;
+drop procedure t_ENUM_CHAR ;
+
+
+call print_message('t_BLOB_CHAR. This scenario is a failure.');
+create or replace procedure t_BLOB_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_BLOB_CHAR('BLOB', 'CHAR', BIT_TO_BLOB(X'000010') ) ;
+drop procedure t_BLOB_CHAR ;
+
+
+call print_message('t_CLOB_CHAR. This scenario is a failure.');
+create or replace procedure t_CLOB_CHAR(sql_type string, procedure_type string, param CHAR ) as begin
+    dbms_output.put_line('sql_type = ' ||sql_type ||', procedure_type = '||procedure_type||', current_value = '|| param ); 
+end;
+call t_CLOB_CHAR('CLOB', 'CHAR', CHAR_TO_CLOB('This is a Dog') ) ;
+drop procedure t_CLOB_CHAR ;
+--BUG
+
+call print_message('t_JSON_CHAR. This scenario is a failure.');
```

Comment:
It's a failure, but the answer file it was successful. Need to check.

Reply:
```suggestion
call print_message('t_JSON_CHAR. This scenario is a success.');
```
Note that it is a success as shown below
```
csql> select cast (json'{"a":1}' as char(10));

=== <Result of SELECT Command in Line 1> ===

   cast(json '{"a":1}' as char(10))
======================
  '{"a":1}   '        

```

Reply:
확인버젼 : CUBRID 11.4 (11.4.0.1339-962675d)

call t_JSON_CHAR('JSON', 'CHAR', json'{"a":1}' ) ;
위와 같이 호출 시에 아래와 같은 에러메세지가 출력됩니다.
ERROR: Stored procedure execute error: unsupported argument type - JSON

PLCSQL에서는 json타입은 미지원함으로 해당 케이스는 정상 처리하도록 하겠으며, 
테스트 comment는 "failure"로 작성하겠습니다.